@xen-orchestra/lite/src/stories/web-core/vts-link.story.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('accent').required().enum('brand', 'success', 'warning', 'danger').preset('brand').widget(),
+      prop('size').required().enum('small', 'medium').preset('medium').widget(),
+      prop('to').type('IconDefinition').widget(text()),
+      prop('href').str().widget(),
+      iconProp(),
+      prop('disabled').bool().widget(),
+      prop('target').enum('_blank', '_self').widget(),
+    ]"
+  >
+    <VtsLink v-bind="properties">This is a link</VtsLink>
+
+    <div v-if="!properties.to && !properties.href" class="info">
+      <UiIcon :icon="faInfoCircle" color="normal" />
```

Comment:
Same as in `VtsLink` component, after a rebase, the icon component could be updated to `VtsIcon` and the color prop to `'brand'`:
```suggestion
      <VtsIcon :icon="faInfoCircle" color="brand" />
```